### DISCUSSION TITLE
WINDOWPLACEMENT

### DISCUSSION THREAD

**Discussion by Michi-S7000** [ControlzEx/ControlzEx#203] [Q&A] [answered]
Hi
In a older Version existed a class ControlzEx.Standard.WINDOWPLACEMENT. Is there a replacement or workaround for this class?
Regards Michael

**Comment by batzen** [comment, marked as answer]
We decided to remove all exposed Win32 classes because they are out of scope to be provided by this library.
We use Microsoft.CsWin32 (https://www.nuget.org/packages/Microsoft.Windows.CsWin32) to generate nearly all interop code.
